Python/Product/Analyzer/Intellisense/OutOfProcProjectAnalyzer.cs

```diff
@@ -1906,7 +1906,7 @@ class TextCodeInfo : CodeInfo {
             if (Path.GetExtension(path).Equals(".xaml", StringComparison.OrdinalIgnoreCase)) {
                 item = _pyAnalyzer.AddXamlFile(path, null);
 
-            } else if (ModulePath.IsPythonSourceFile(path)) {
```

Comment:
It's ok to do this for non-python files now?

Reply:
Yeah, we'll just treat it like a source file. Basically, we special-case .XAML, and if you tell us to analyze another file type then we'll assume it's source code.

Reply:
The trick is going to be in not adding all files to the analyzer, but that'll be on the VS side. This way we can tie it entirely to other options, such as the build action being "Compile".

Reply:
Ok, for now as long as it doesn't crash or cause major issues with text files, etc.